tests/Amazon.CloudWatch.EMF.Tests/Model/MetricDefinitionTests.cs

```diff
@@ -43,13 +43,23 @@ public void MetricDefinition_WithUnitAndValue_Returns_ValidJson()
         [Fact]
         public void MetricDefinition_AddValue_Returns_ValidValues()
         {
-            MetricDefinition metricDefinition = new MetricDefinition("Time", Unit.MILLISECONDS, 10);
+            MetricDefinition metricDefinition = new MetricDefinition("Time", 10, Unit.MILLISECONDS);
 
             Assert.Equal(new List<double>() { 10 }, metricDefinition.Values);
 
             metricDefinition.AddValue(20);
 
             Assert.Equal(new List<double>() { 10, 20 }, metricDefinition.Values);
         }
+
+        [Fact]
+        public void MetricDefinition_WithHighResolution_Returns_ValidJson()
+        {
+             MetricDefinition metricDefinition = new MetricDefinition("Time", 10, Unit.MILLISECONDS,StorageResolution.HIGH);
+             
+             string metricString = JsonConvert.SerializeObject(metricDefinition);
+             
+             Assert.Equal("{\"Name\":\"Time\",\"Unit\":\"Milliseconds\",\"StorageResolution\":\"1\"}", metricString);
+        }
```

Comment:
Can we have additional tests for these metric definitions:

MetricDefinition("Time", 10, StorageResolution.HIGH)
MetricDefinition("Time", 10, StorageResolution.STANDARD)

(putMetric tests without `unit` and with `storageResolution`)

Reply:
Done